### DISCUSSION TITLE
How to set focus to a selectOneMenu component

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#112] [PrimeFaces] [answered]
In version 11 there was an extra input element (_focus) in the html markup for selectOneMenu to be able to set the focus. In PF13 this element no longer exists. How can I now set the focus on a selectOneMenu component?

**Comment by tandraschko** [comment, marked as answer]
the `label for="..."` now just targets `_label`, i would try that

**Reply by hgfrey** [reply]
FIne. That works :-)
